odoo/models.py

```diff
@@ -4361,6 +4362,7 @@ def _generate_m2o_order_by(self, alias, order_field, query, reverse_direction, s
     def _generate_order_by_inner(self, alias, order_spec, query, reverse_direction=False, seen=None):
         if seen is None:
             seen = set()
+        order_spec = urls.url_unquote(order_spec)
```

Comment:
Could you do that in the forum controller (before `Post._generate_order_by(sorting, None)`)?
While this is not wrong, this is not something we want to add in that low level method, used by the ORM. That would "pollute" such low level business code not supposed to be handling URL encoded params.
That's really the job of the caller here, not the callee.